src/main/scala/workbench/Plugin.scala

```diff
@@ -134,21 +130,19 @@ object Plugin extends sbt.Plugin {
             Some(output.getParentFile.toURI())
           else None
 
-        import ScalaJSOptimizer._
-
-        (scalaJSOptimizer in fastOptJS).value.optimizeCP(
```

Comment:
I had some trouble here, as this part has since been refactored in scala-js. It seems to work while this is commented out, but do you have any idea of what the intention here was? disable cache?